automated-comments/java/hamming/calculate_distance_in_constructor.md

```diff
@@ -0,0 +1,3 @@
+This solution seems to be calculating the Hamming distance inside the `getHammingDistance()` method.
+While this is acceptable, it means that the calculation might be done again if you did not use lazy initialization.
```

Comment:
Possibly rephrase without `While this is acceptable,`

See guidelines about "This is perfectly fine" example.